test/check-docker

```diff
@@ -100,13 +101,14 @@ class TestDocker(MachineCase):
         b.click('#containers-containers tr:contains("PROBE")')
         b.wait_page("container-details")
         b.wait_in_text("#container-terminal", "Hello from container-probe.")
-        b.call_js_func("ph_count_check", ".logs",  1)
+        self.assertTrue(b.call_js_func("ph_count_check", ".logs",  1))
```

Comment:
This doesn't look correct (but is likely harmless).

ph_count_check already throws an exception (which gets turned into a Python exception) if the counts don't match, so the assertion is superflous. Also, ph_count_check returns the count, not a boolean.